firebase.json

```diff
@@ -203,9 +203,10 @@
       { "source": "/resources/dart-tips*{,/**}", "destination": "/guides/language/effective-dart", "type": 301 },
       { "source": "/resources/dartpad-best-practices", "destination": "/tools/dartpad/dartpad-best-practices", "type": 301 },
       { "source": "/resources/dartpad-tutorials.pdf", "destination": "/resources/dartpad-best-practices", "type": 301 },
-      { "source": "/samples/spirodraw{,/**}", "destination": "/samples", "type": 301 },
-      { "source": "/samples/todomvc{,/**}", "destination": "/samples", "type": 301 },
-      { "source": "/samples/video{,/**}", "destination": "/samples", "type": 301 },
+      { "source": "/samples/", "destination": "/language", "type": 301 },
+      { "source": "/samples/spirodraw{,/**}", "destination": "/language", "type": 301 },
+      { "source": "/samples/todomvc{,/**}", "destination": "/language", "type": 301 },
+      { "source": "/samples/video{,/**}", "destination": "/language∏SAVE", "type": 301 },
```

Comment:
What's the character in the destination here, `/language∏SAVE`? It's giving me a 404

Reply:
It was a bug! Fixed now.